src/targets/__tests__/symbolCollector.test.ts

```diff
@@ -0,0 +1,138 @@
+import { withTempDir } from '../../utils/files';
+import { NoneArtifactProvider } from '../../artifact_providers/none';
+import { checkExecutableIsPresent, spawnProcess } from '../../utils/system';
+import { SymbolCollector, SYM_COLLECTOR_BIN_NAME } from '../symbolCollector';
+
+jest.mock('../../utils/files');
+jest.mock('../../utils/system');
+jest.mock('fs', () => {
```

Comment:
We may wanna lift the common `fs` mocks into a global mock module as this pattern seems common enough now.